packages/iov-bns/src/decode.ts

```diff
@@ -224,13 +224,13 @@ function parseSwapClaimTx(
   };
 }
 
-function parseSwapTimeoutTx(
+function parseSwapAbortTransaction(
```

Comment:
Do we have any rules about when to use `tx` and when to use `transaction`? This change is not consistent with how it was before or the `parseSwapClaimTx` function.

Reply:
My idea is to ban the term "tx" in all non-Tendermint code. It seems to be very popular in the Tendermint community but not necessarily in all blockchain world. This is why in BCP everything is *transaction but there is still a lot of legacy.

The whole naming build* and parse* in encode.ts/decode.ts is inconsitent. It should probably be "encode<transaction type>" and "decode<transaction type>". I started this renaming as part of this PR but it became too much so I stopped the internal renamings to focus on external facing changes only.

Reply:
OK, I'm happy to incrementally switch to `transaction` only.